### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by CyberShadow**
Fix Markdown list indentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,12 +56,12 @@ We are hoping to collect a bunch of themes for our command prompts. You can see
 ### Manual installation
 
 1. Clone the repository
-```fish
+       ```fish
 git clone git://github.com/bpinto/oh-my-fish.git ~/.oh-my-fish
```
Comment (bobthecow) [inline README.md]: if you're going to indent the block, indent the whole block. this is just gross ;)

Comment (CyberShadow) [inline README.md]: That would add the leading spaces to the code being indented, thus indenting it twice.

Comment (bobthecow) [inline README.md]: Oh, the joys of markup languages without a spec. It doesn't on commonmark (and some other fenced code block implementations).

Comment (CyberShadow) [inline README.md]: Never mind. You're right.

**Comment by bucaran** [comment]
:+1: Thanks @CyberShadow

**Pull request merged**